batch_processing.sh

```diff
@@ -126,11 +126,23 @@ cd ..
 # t1
 # ===========================================================================================
 cd t1
-# Segment spinal cord
-sct_deepseg_sc -i t1.nii.gz -c t1 -qc "$SCT_BP_QC_FOLDER"
-# Smooth spinal cord along superior-inferior axis
+# Contrast-agnostic registration
+# 1. t1w preprocessing (cropping around spinal cord)
+sct_deepseg_sc -i t1.nii.gz -c t1 -centerline cnn
+sct_create_mask -i t1.nii.gz -p centerline,t1_seg.nii.gz -size 35mm -f cylinder -o mask_t1.nii.gz
+sct_crop_image -i t1.nii.gz -m mask_t1.nii.gz
+# 2. t2w preprocessing (cropping around spinal cord)
+cp ../t2/t2.nii.gz ./t2.nii.gz
+sct_deepseg_sc -i t2.nii.gz -c t2
+sct_create_mask -i t2.nii.gz -p centerline,t2_seg.nii.gz -size 35mm -f cylinder -o mask_t2.nii.gz
+sct_crop_image -i t2.nii.gz -m mask_t2.nii.gz
+# 3. Perform registration
+sct_register_multimodal -i t1_crop.nii.gz -d t2_crop.nii.gz -param step=1,type=im,algo=dl
```

Comment:
I've added the registration pipeline for T1/T2 data in `sct_example_data`. Here are some comparison screenshots:

`t1_crop_reg.nii.gz` + `t2_crop.nii.gz`:

![ezgif-3-8492cdd8c6](https://user-images.githubusercontent.com/16181459/175089606-c4615a4d-2a98-4352-b1fe-e900c5bb1f3a.gif)

Zoomed in, with increased contrast on the T1 image:

![ezgif-3-8e868e9e6e](https://user-images.githubusercontent.com/16181459/175089601-2ae5bd55-f447-4edd-97e9-688e74d0f7d7.gif)

Reply:
this is amazing! the QC you performed/documented is excellent (i wouldn't have done a better job)